crates/arroyo-connectors/src/nats/table.json

```diff
@@ -0,0 +1,179 @@
+                                        "stream": {
+                                            "title": "Stream Name",
+                                            "type": "string",
+                                            "description": "The stream the consumer should consume from."
+                                        },
+                                        "ackPolicy": {
+                                            "title": "Acknowledgment Policy",
+                                            "type": "string",
+                                            "description": "How messages should be acknowledged.",
+                                            "enum": [
+                                                "Explicit",
+                                                "None",
+                                                "All"
+                                            ],
+                                            "default": "Explicit"
+                                        },
+                                        "replayPolicy": {
+                                            "title": "Replay policy",
+                                            "type": "string",
+                                            "description": "Whether messages are sent as quickly as possible or at the rate of receipt.",
+                                            "enum": [
+                                                "Original",
+                                                "Instant"
+                                            ],
+                                            "default": "Instant"
+                                        },
+                                        "ackWait": {
+                                            "title": "Acknowledgment wait",
+                                            "type": "integer",
+                                            "description": "The duration that the server will wait for an ack for any individual message once it has been delivered to a consumer.",
+                                            "default": 300
+                                        },
+                                        "description": {
+                                            "title": "Consumer description",
+                                            "type": "string",
+                                            "description": "A description of the consumer. This can be particularly useful for ephemeral consumers to indicate their purpose since the durable name cannot be provided."
+                                        },
+                                        "filterSubjects": {
+                                            "title": "Filter subjects",
+                                            "type": "string",
+                                            "description": "A list of subjects that the consumer should filter on. If the list is empty, then no filtering is done.",
```

Comment:
I'd specify the format for this, whether it's comma-separated or what. You can also use an array which will render a nice UI.

Reply:
Now rendered as an array. The enumerated items descriptions are set to `undefined_n` though, not sure how to fix that?